cookbooks/aws-parallelcluster-scheduler-plugin/recipes/init.rb

```diff
@@ -15,6 +15,15 @@
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Set the HOME environment variable to the home directory of the target user
+cookbook_file "1_parallelcluster" do
+  source "sudo_set_home/1_parallelcluster"
+  path "/etc/sudoers.d/1_parallelcluster"
```

Comment:
I think we have already a pcluster file in sudoers.d dir, can we reuse that?

Reply:
See https://github.com/aws/aws-parallelcluster-cookbook/tree/develop/cookbooks/aws-parallelcluster-scheduler-plugin/templates/default/scheduler_plugin_user

Reply:
Done, added to 99-parallelcluster-scheduler-plugin.erb